rtc-video-quality/generate_graphs.py

```diff
@@ -116,15 +116,15 @@ def generate_stt(data, output_dir=''):
 
 
         filtered_item = list(filter(lambda item: item['encoder'] == encoder and item['codec'] == codec and item['input-file'] == video and bitrate == item['actual-bitrate-bps'], data))
-        assert len(filtered_item ) == 1
-        item = filtered_item[0]
-
-        ## Filter my dictionary to contain the required key values
-        required_data = OrderedDict()
-        required_data['bitrate'] = str(bitrate)
-        for key in metrics:
-          if key in item:
-            required_data[key] = "{:.2f}".format(item[key])
+            
+        for item in filtered_item:
```

Comment:
Can we add a warning if this occurs (Same actual bitrate with two different target bitrates)? That way, users can know to pick better or different bitrate targets.

Reply:
Yeah sure

Reply:
Updated https://github.com/googleinterns/av1-codec-comparison/pull/20/commits/4b5ebe959a37f6f584d865869f705f207e4740f8